config/locales/en.yml

```diff
@@ -1797,6 +1812,7 @@ en:
     no_login: "This user authenticates through login by password. Since it is disabled, they cannot log in."
     password_change_unsupported: Change of password is not supported.
     authorization_rejected: "You are not allowed to sign in."
+    invite: Invite user via e-mail
```

Comment:
You've used `email` and `e-mail`, I think [both are fine in english](http://dictionary.cambridge.org/dictionary/english/email), but keep it consistent.